### PR TITLE
io.gitlab.Goodvibes

### PR DESCRIPTION
**Pull request by elboulangero**

### CODE REVIEW

File: io.gitlab.Goodvibes.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+  - name: goodvibes
+    buildsystem: meson
+    builddir: true
+    sources:
+      - type: git
+        url: https://gitlab.com/goodvibes/goodvibes.git
+        tag: v0.4.2
+        commit: 7559cfc0291871fa05acadb4ff3f19a4a89c997c
+cleanup:
```
Comment (x80486) [inline io.gitlab.Goodvibes.yaml]: NIT: This usually goes after `finish-args` :innocent:

File: io.gitlab.Goodvibes.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  - name: keybinder
+    sources:
+      - type: archive
+        url: https://github.com/kupferlauncher/keybinder/releases/download/keybinder-3.0-v0.3.2/keybinder-3.0-0.3.2.tar.gz
+        sha256: e6e3de4e1f3b201814a956ab8f16dfc8a262db1937ff1eee4d855365398c6020
+    cleanup: [ /include, /lib/*.la, /lib/girepository-1.0, /lib/pkgconfig, /share ]
+
+  - name: goodvibes
+    buildsystem: meson
+    builddir: true
```
Comment (TingPing) [inline io.gitlab.Goodvibes.yaml]:
```suggestion
```

This is the default for meson.

**Comment by TingPing** [comment]
bot, build io.gitlab.Goodvibes

**Comment by TingPing** [comment]
Upstream appdata file looks good. You can add OARS data: https://hughsie.github.io/oars/generate.html (it will just be an empty tag)

File: io.gitlab.Goodvibes.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  - '--env=DCONF_USER_CONFIG_DIR=.config/dconf'
+  # MPRIS2 support
+  - '--own-name=org.mpris.MediaPlayer2.Goodvibes'
+modules:
+  - name: keybinder
+    sources:
+      - type: archive
+        url: https://github.com/kupferlauncher/keybinder/releases/download/keybinder-3.0-v0.3.2/keybinder-3.0-0.3.2.tar.gz
+        sha256: e6e3de4e1f3b201814a956ab8f16dfc8a262db1937ff1eee4d855365398c6020
+    cleanup: [ /include, /lib/*.la, /lib/girepository-1.0, /lib/pkgconfig, /share ]
```
[the remark this answers is not in the document]
Comment (elboulangero) [reply io.gitlab.Goodvibes.yaml]: Agreed, fixed in c8f8185, thx!

File: io.gitlab.Goodvibes.yaml (hunk 1 above)
Comment (elboulangero) [reply io.gitlab.Goodvibes.yaml]: OK, no worries, I moved the `cleanup` field up there after `finish-args` in 5afe9e6. Thanks!

File: io.gitlab.Goodvibes.yaml (hunk 2 above)
Comment (elboulangero) [reply io.gitlab.Goodvibes.yaml]: Alright, removed in e70c467. Thanks!

**Comment by elboulangero** [comment]
> You can add OARS data: https://hughsie.github.io/oars/generate.html

I just did that, thanks for pointing the way :) It's indeed an empty tag

**Comment by nedrichards** [comment]
bot, build io.gitlab.Goodvibes

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/io.gitlab.Goodvibes

**Pull request closed without merging**